gradle.properties

```diff
@@ -4,6 +4,9 @@ org.gradle.jvmargs=-Xmx2g
 springBootVersion = 2.7.2
 graphglueVersion = 4.1.0
 graphqlJavaVersion = 18.1
+apolloVersion = 3.3.0
```

Comment:
if possible, update to newest dependency version

Reply:
f3c3dad7f2c5c2dd4a8da5ce86a345cb7d6acfae